pennylane/pulse/transmon.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains the classes/functions specific for simulation of superconducting transmon hardware systems"""
+from dataclasses import dataclass
+from typing import Callable, List, Union
+
+import pennylane as qml
+from pennylane.pulse import HardwareHamiltonian
+from pennylane.typing import TensorLike
+
+
+# TODO ladder operators once there is qudit support
+# pylint: disable=unused-argument
+def a(wire, d=2):
+    """creation operator"""
+    return qml.s_prod(0.5, qml.PauliX(wire)) + qml.s_prod(0.5j, qml.PauliY(wire))
+
+
+def ad(wire, d=2):
+    """annihilation operator"""
+    return qml.s_prod(0.5, qml.PauliX(wire)) + qml.s_prod(-0.5j, qml.PauliY(wire))
+
+
+# pylint: disable=too-many-arguments
+def transmon_interaction(
+    omega: Union[float, list],
+    connections: list,
+    g: Union[float, list],
+    anharmonicity=None,
+    wires=None,
+    d=2,
+):
+    r"""Returns a :class:`ParametrizedHamiltonian` representing the cQED Hamiltonian of a superconducting transmon system.
+
+    The Hamiltonian is given by
+
+    .. math::
+
+        H = \sum__{q\in \text{wires}} \omega_q a^\dagger_q a_q
```

Comment:
This is rendering as a weird text box, I'm not sure if this is why, but something is up
```suggestion
        H = \sum_{q\in \text{wires}} \omega_q a^\dagger_q a_q
```